runtimes/dotnet/dotnet.md

```diff
@@ -0,0 +1,123 @@
+
+## Create an application
+
+Refer to the page [Deploy an application on Clever Cloud](/doc/clever-cloud-overview/add-application/).
+
+## Necessary information
+
+Be sure that:
+
+* you have pushed in <b>master branch</b>
+* you listen on <b>port 8080</b>
+* you have committed the different files of your project and the corresponding `.csproj` file
+
+## Requirements
+
+Let's take an example with the Microsoft [TodoApi project](https://github.com/Azure-Samples/dotnet-core-api.git). 
+
+During deployment, the `TodoApi.csproj` file and the target framework `netcoreapp3.1` are automatically detected. Then, the .NET project is published :
+```sh
+dotnet publish --framework netcoreapp3.1 --configuration Release
+```
+
+No additional configuration is required (unless multiple .csproj files or target frameworks are present, see the documentation below).
+
+### Multiple csproj files in the repository
+
+If multiple `.csproj` files are present in your repository, you can specify the file to use with the 
+`CC_DOTNET_CSPROJ` environment variable.
+```sh
+CC_DOTNET_CSPROJ=TodoApi2.csproj
+```
+
+### Multiple binary targets
+
+If your `.csproj` file defines multiple targets, like :
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0;netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+  ...
+```
+You must specify the one you want to run, with the `CC_DOTNET_TFM` environment variable. If `CC_DOTNET_TFM` is
+specified, then the executable produced by this target is used to start the
+application.
+```sh
+CC_DOTNET_TFM=netcoreapp3.1
+```
+
+### Dependencies
+
+Make sure to list all your dependencies in your `.csproj` file. For example:
+
+```xml
+  ...
+  <ItemGroup>
+    <PackageReference Include="EntityFramework" Version="6.4.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.2" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.3">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>
+</Project>
+```
+
+Compiled dependencies are cached by default to speed up deployments. You can
+disable dependencies caching completely by removing the `CC_CACHE_DEPENDENCIES`
+environment variable. If you want to rebuild your application from scratch,
+you can select "rebuild and restart" from the console or launch `clever
+restart --without-cache` from CLI.
+
+### The configuration to publish for
+`Release` by default, you can use the `CC_DOTNET_PROFILE` environment variable to change this configuration.
+```sh
+CC_DOTNET_PROFILE=Debug
+```
+
+### Private dependencies
+
+Support for private dependencies will be available soon.
+
+### Dotnet version
+
+Only version 3.1 is available at the moment (`3.1.106`). It will be possible to choose different versions in the future
```

Comment:
Not sure it's worth it to specity the exact micro version, it's easy to forget to update this specific bit when updating it